libraries/AdaptiveExpressions/ExpressionFunctions.cs

```diff
@@ -3483,7 +3483,7 @@ private static IEnumerable<object> Flatten(IEnumerable<object> list, int dept)
                             }
                             else
                             {
-                                error = $"{expr} can't evaluate.";
+                                error = $"{expr} should contain an ISO format timestamp, an integer of time interval, a string represents unit of time and an optional string of output format.";
```

Comment:
>should contain an ISO format timestamp, an integer of time interval, a string represents unit of time and an optional string of output format [](start = 49, length = 141)

This is awkward.  How about "should contain an ISO format timestamp, a time interval integer, a string unit of time and an optional output format string."

Reply:
changed